wallet/wallet.go

```diff
@@ -24,6 +24,16 @@ const (
 	TxnSourceFoundationPayout TransactionSource = "foundation"
 )
 
+const (
+	// bytesPerInput is the encoded size of a SiacoinInput and corresponding
+	// TransactionSignature, assuming standard UnlockConditions.
+	bytesPerInput = 241
```

Comment:
@lukechampine should this eventually be exposed by `core`?

Reply:
I don't think so. It doesn't even really need to be exact; it's just a helper for calculating a reasonable txn fee.